datasetcard.md

```diff
@@ -0,0 +1,118 @@
+- {size_0}  # Example: 100K<n<1M
+source_datasets:
+- {source_dataset_0}  # Example: wikipedia
+- {source_dataset_1}  # Example: laion/laion-2b
+task_categories:  # Full list at https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts
+- {task_0}  # Example: question-ansering
+- {task_1}  # Example: image-classification
+task_ids:
+- {subtask_0}  # Example: extractive-qa
+- {subtask_1}  # Example: multi-class-image-classification
+paperswithcode_id: {paperswithcode_id}  # Dataset id on PapersWithCode (from the URL). Example for SQuAD: squad
+configs:  # Optional for datasets with multiple configurations like glue.
+- {config_0}  # Example for glue: sst2
+- {config_1}  # Example for glue: cola
+
+# Optional. This part can be used to store the feature types and size of the dataset to be used in python. This can be automatically generated using the datasets-cli.
+dataset_info:
+  features:
+    - name: {feature_name_0}    # Example: id
+      dtype: {feature_dtyp_e0}  # Example: int32
+    - name: {feature_name_1}    # Example: text
+      dtype: {feature_dtype_1}  # Example: string
+    - name: {feature_name_2}    # Example: image
+      dtype: {feature_dtype_2}  # Example: image
+    # Example for SQuAD:
+    # - name: id
+    #   dtype: string
+    # - name: title
+    #   dtype: string
+    # - name: context
+    #   dtype: string
+    # - name: question
+    #   dtype: string
+    # - name: answers
+    #   sequence:
+    #     - name: text
+    #       dtype: string
+    #     - name: answer_start
+    #       dtype: int32
+  config_name: {config_name}  # Example for glue: sst2
+  splits:
+    - name: {split_name_0}                  # Example: train
+      num_bytes: {split_num_bytes_0}        # Example for SQuAD: 79317110
+      num_examples: {split_num_examples_0}  # Example for SQuAD: 87599
+  download_size: {dataset_download_size}   # Example for SQuAD: 35142551
+  dataset_size: {dataset_size}             # Example for SQuAD: 89789763
+
+# It can also be a list of multiple configurations:
+# ```yaml
+# dataset_info:
+#   - config_name: {config0}
+#     features:
+#       ...
+#   - config_name: {config1}
+#     features:
+#       ...
+# ```
+
+# Optional. If you want your dataset to be protected behind a gate that users have to accept to access the dataset:
+extra_gated_fields:
+- {field_name_0}: {field_type_0}  # Example: Name: text
+- {field_name_1}: {field_type_1}  # Example: Affiliation: text
+- {field_name_2}: {field_type_2}  # Example: Email: text
+- {field_name_3}: {field_type_3}  # Example for speech datasets: I agree to not attempt to determine the identity of speakers in this dataset: checkbox
+extra_gated_prompt: {extra_gated_prompt}  # Example for speech datasets: By clicking on “Access repository” below, you also agree to not attempt to determine the identity of speakers in the dataset.
+
+# Optional. Add this if you want to encode a train and evaluation info in a structured way for AutoTrain
```

Comment:
Any doc you can link for this?

Reply:
I haven't found any doc about `train-eval-index`. Maybe @lewtun @abhishekkrthakur know more